code/datums/ability.dm

```diff
@@ -299,6 +314,27 @@
 	if(always_bind)
 		quickbind()
 
+
+/**
+ * checks if the target is within range and the ability is triggerable
+ */
+/datum/ability/proc/target_check(mob/user, atom/target)
+	if(range)
+		if(get_dist(get_turf(user),get_turf(target)) <= range)
```

Comment:
whats the point of this check

do you mean if not then return false rather than if else return false